### PR TITLE
cloud experience: add awscli2 and related pkgs

### PR DESCRIPTION
**Pull request by major**

### CODE REVIEW

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: configs/sst_cloud_experience.yaml [hunk 1]
```diff
@@ -7,7 +7,19 @@ data:
 
   packages:
     - efs-utils
+    - python3-awscrt
     - python3-botocore
+    - python3-prompt-toolkit
+    - python3-ruamel-yaml-clib
+    - python3-websockets
```
[the remark this answers is not in the document]
Comment (major) [reply configs/sst_cloud_experience.yaml:14]: That should be it for now. I have this building in a COPR and that's all we need. There's one missing, `python-flit-core`, but I'm checking with the Python Maint team about them taking that one.

[the remark this answers is not in the document]
Comment (yselkowitz) [reply configs/sst_cloud_experience.yaml:13]: python-flit-core is already in the buildroot, so no worries there.

File: configs/sst_cloud_experience.yaml [hunk 2]
```diff
@@ -8,6 +8,20 @@ data:
   packages:
     - efs-utils
     - python3-botocore
+    - python3-prompt-toolkit
+    - python3-ruamel-yaml-clib
```
Comment (yselkowitz) [inline configs/sst_cloud_experience.yaml:12]:
```suggestion
    - python3-ruamel-yaml
    - python3-ruamel-yaml-clib
```
These come from separate SRPMs and both are required.

**Comment by yselkowitz** [comment]
headsup @davide125 wrt awscli2 currently in eln_extras_meta

File: configs/sst_cloud_experience.yaml (hunk 2 above)
Comment (major) [reply configs/sst_cloud_experience.yaml:13]: Fixed.

**Review by yselkowitz — APPROVED** [review]

**Comment by hroncok** [comment]
awscli2 now pulls in pytest-xdist into RHEL. See https://github.com/fedora-eln/eln/issues/125

**Pull request merged**